From 7bf5a81dfb104652abd3cfc60d1b0679db9a7a94 Mon Sep 17 00:00:00 2001
From: Gaurav Vohra <gauravx.vohra@intel.com>
Date: Sun, 21 Jul 2019 23:37:10 +0800
Subject: [PATCH] ASoC: Intel: Common: IVI(RSE,HU,GENERIC) Addition of ACPI IDs
 for Machine Driver loading

Addition of ACPI IDs for Machine Driver loading. Changes made to sound/soc/intel/common/soc-acpi-intel-bxt-match.c

Signed-off-by: Gaurav Vohra <gauravx.vohra@intel.com>
---
 sound/soc/intel/common/soc-acpi-intel-bxt-match.c | 20 ++++++++++++--------
 1 file changed, 12 insertions(+), 8 deletions(-)

diff --git a/sound/soc/intel/common/soc-acpi-intel-bxt-match.c b/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
index c7d6057..8445754 100644
--- a/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
+++ b/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
@@ -20,25 +20,29 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_bxt_machines[] = {
 		.id = "INT343A",
 #if IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_RSE_MACH)
 		.drv_name = "bxt_ivi_rse_i2s",
-#elif IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_HU_MACH)
-		.drv_name = "bxt_ivi_hu_i2s",
 #elif IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_M3_MACH)
 		.drv_name = "bxt_ivi_m3_i2s",
-#elif IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_GENERIC_MACH)
-		.drv_name = "bxt_ivi_generic_i2s",
 #else
 		.drv_name = "bxt_alc298s_i2s",
 #endif
 #if IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_RSE_MACH) || \
-	IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_HU_MACH) || \
-	IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_M3_MACH) || \
-	IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_GENERIC_MACH)
-		.fw_filename = "intel/ADSPFW.bin"
+	IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_M3_MACH)
+		.fw_filename = "dsp_fw_release.bin"
 #else
 		.fw_filename = "intel/dsp_fw_bxtn.bin",
 #endif
 	},
 	{
+		.id = "INT344A",
+		.drv_name = "bxt_ivi_hu_i2s",
+		.fw_filename = "dsp_fw_release.bin",
+	},
+	{
+		.id = "INT344B",
+		.drv_name = "bxt_ivi_generic_i2s",
+		.fw_filename = "dsp_fw_release.bin",
+	},
+	{
 		.id = "INT345A",
 		.drv_name = "lhcrb_wm8731_i2s",
 		.fw_filename = "dsp_fw_release.bin",
-- 
1.9.1

